### PR TITLE
web: Include authors on each page

### PR DESCRIPTION
**Pull request by SquidDev**
Adds a "Written by: xyz" section to each page:

A screenshot of 1Lab.Path, with 'Written by Amélia Liao, Astra Kolomatskaia, Jonathan Coates and uni' on the bottom left.

It's a little dubious (git log isn't smart enough to exclude whitespace changes), but good enough. I think!

### CODE REVIEW

**Comment by plt-amy** [comment]
🎉

File: src/index.lagda.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ principle depends on [_universes_](agda://1Lab.Type), [_propositional
 equality_](agda://1Lab.Path) (paths) and [_equivalences_](agda://1Lab.Equiv). 
 In addition to the hyperlinked "web of concepts" provided by the Agda
 code, there is a short introduction to homotopy type theory: **[Start
-here](1Lab.intro.html)**.
+here](1Lab.Intro.html)**.
```
Comment (plt-amy) [inline src/index.lagda.md:15]: What happened here?

**Review by plt-amy — CHANGES_REQUESTED** [review]

File: src/index.lagda.md (hunk 1 above)
Comment (SquidDev) [reply src/index.lagda.md:15]: I also noticed the link to 1lab instead of 1Lab, but fixed the wrong thing instead. I cannot explain myself.

Comment (plt-amy) [reply src/index.lagda.md:15]: haha

**Pull request merged**